x/wasm/keeper/handler_plugin_test.go

```diff
@@ -270,19 +285,39 @@ func TestIBCRawPacketHandler(t *testing.T) {
 		},
 	}
 	contractKeeper := wasmtesting.IBCContractKeeperMock{}
+	// also store a packet to be acked
+	ackPacket := channeltypes.Packet{
+		Sequence:           1,
+		SourcePort:         "src-port",
+		SourceChannel:      "channel-0",
+		DestinationPort:    ibcPort,
+		DestinationChannel: "channel-1",
+		Data:               []byte{},
+		TimeoutHeight:      clienttypes.Height{},
+		TimeoutTimestamp:   0,
```

Comment:
I think one of them must be set to a non-empty value, but not sure

Reply:
For actual data, one of them should be set, but here we just test that WriteAcknowledgement is called with the same packet that was saved earlier, so what that packet looks like is not that relevant, except for the parts we care about (destination port, channel and sequence)